### ISSUE TITLE
[New rule] error when rule name not found

### ISSUE THREAD

**Issue by DetachHead** [dart-code-checker/dart-code-metrics#1051]
**Please describe what the rule should do:**

if there's an invalid rule name in `analysis_options.yaml`, it should cause an error

**What category of rule is this? (place an "X" next to just one item)**

- [X] Warns about a potential error (problem)
- [ ] Suggests an alternate way of doing something (suggestion)
- [ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about (it will be better if you can provide both good and bad examples):**

```yaml
dart_code_metrics:
  rules:
    - asdf # error
    - no-empty-block # no error
```

**Are you willing to submit a pull request to implement this rule?**
idk

**Comment by incendial** [comment]
That's planned for the next release, stay tuned 😉

**Comment by incendial** [comment]
Available in 5.0.0 🚀
